### PR TITLE
Fix copyright year 2021

### LINKED ISSUE #187: copyright year 2021
image

### PR DESCRIPTION
**Pull request by lrhel**
fixes #187

### CODE REVIEW

File: doc/man/sugarchain-cli.1 [hunk 1]
```diff
@@ -100,7 +100,7 @@ Copyright (C) 2009-2010 Satoshi Nakamoto
 Copyright (C) 2009-2018 The Bitcoin Core developers
 Copyright (C) 2013-2019 Alexander Peslyak - Yespower 1.0.1
 Copyright (C) 2016-2018 The Zcash developers - DigiShieldZEC
-Copyright (C) 2018-2020 The Sugarchain Yumekawa developers
+Copyright (C) 2018-2021 The Sugarchain Yumekawa developers
```
Comment (decryp2kanon) [inline doc/man/sugarchain-cli.1]: this man help should be automatically generated. no need to edit here

File: doc/man/sugarchain-qt.1 [hunk 2]
```diff
@@ -569,7 +569,7 @@ Copyright (C) 2009-2010 Satoshi Nakamoto
 Copyright (C) 2009-2018 The Bitcoin Core developers
 Copyright (C) 2013-2019 Alexander Peslyak - Yespower 1.0.1
 Copyright (C) 2016-2018 The Zcash developers - DigiShieldZEC
-Copyright (C) 2018-2020 The Sugarchain Yumekawa developers
+Copyright (C) 2018-2021 The Sugarchain Yumekawa developers
```
Comment (decryp2kanon) [inline doc/man/sugarchain-qt.1]: this man help should be automatically generated. no need to edit here

File: doc/man/sugarchain-tx.1 [hunk 3]
```diff
@@ -116,7 +116,7 @@ Copyright (C) 2009-2010 Satoshi Nakamoto
 Copyright (C) 2009-2018 The Bitcoin Core developers
 Copyright (C) 2013-2019 Alexander Peslyak - Yespower 1.0.1
 Copyright (C) 2016-2018 The Zcash developers - DigiShieldZEC
-Copyright (C) 2018-2020 The Sugarchain Yumekawa developers
+Copyright (C) 2018-2021 The Sugarchain Yumekawa developers
```
Comment (decryp2kanon) [inline doc/man/sugarchain-tx.1]: this man help should be automatically generated. no need to edit here

File: doc/man/sugarchaind.1 [hunk 4]
```diff
@@ -548,7 +548,7 @@ Copyright (C) 2009-2010 Satoshi Nakamoto
 Copyright (C) 2009-2018 The Bitcoin Core developers
 Copyright (C) 2013-2019 Alexander Peslyak - Yespower 1.0.1
 Copyright (C) 2016-2018 The Zcash developers - DigiShieldZEC
-Copyright (C) 2018-2020 The Sugarchain Yumekawa developers
+Copyright (C) 2018-2021 The Sugarchain Yumekawa developers
```
Comment (decryp2kanon) [inline doc/man/sugarchaind.1]: this man help should be automatically generated. no need to edit here

**Review by decryp2kanon — CHANGES_REQUESTED** [review]
this man help should be automatically generated. no need to edit here

```
.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.8.
```

```bash
make -j$(nproc) && ./contrib/devtools/gen-manpages.sh
```

**Comment by lrhel** [comment]
my bad, uncommitted the man pages

**Comment by decryp2kanon** [comment]
closing due to too old

**Pull request closed without merging**